variables.tf

```
@@ -144,6 +144,27 @@ variable "managed_rules" {
   default = {}
 }
 
+variable "recording_mode" {
+  description = <<-DOC
+    The mode for AWS Config to record configuration changes. Possible values are:
+    - Continuous
+    - Daily
```

Provider docs seem to uppercase these, so just in case that is an issue..

```suggestion
    - CONTINUOUS
    - DAILY
```